docs/spec.md

```diff
@@ -58,11 +58,11 @@ At the moment, StableHLO only supports dense tensors, so each tensor has
 `1*(d0)*(d1)*...*(dR-1)` elements whose indices are drawn from an
 **index space** which is a Cartesian product of its dimensions. For example:
 
-  * `tensor<2x3xf32>` has 6 elements whose indices are
+* `tensor<2x3xf32>` has 6 elements whose indices are
     `{0, 0}`, `{0, 1}`, `{0, 2}`, `{1, 0}`, `{1, 1}` and `{1, 2}`.
-  * Tensors of rank zero, e.g `tensor<f32>`, have 1 element. Such tensors are
+* Tensors of rank zero, e.g `tensor<f32>`, have 1 element. Such tensors are
     allowed and are useful to model scalars.
-  * Tensors with dimensions of size zero, e.g. `tensor<2x0xf32>`, have
+* Tensors with dimensions of size zero, e.g. `tensor<2x0xf32>`, have
     0 elements. Such tensors are allowed and are useful in rare cases, e.g.
     to model empty slices.
```

Comment:
Should indentation of lines 66 and 67 (as well as some lines above) be changed to align with the first line of the bullet? (Same for lines 605-619, 679-693, 736, 1032, 1115, 1203, 1234-1249, 1270, 1323, 1570-1583, 1611, 1677-1695, 2258-2260, 2270-2282, and more).

I understand that this is non-trivial work to align these indentations, so I can do it on my own too, just wondering about the canonical rules for indenting multiline bullets.

Reply:
Dang, I didn't notice this. It was caused by the auto-fix for list items that were unnecessarily indented ([rule MD007](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md007---unordered-list-indentation)), but the autofix does not remove indentation from the wrapped lines. Perhaps I should revert that part of the change (fortunately it was a separate commit) and then we can manually respond to all the MD007 errors manually, so that we're sure to also un-indent the wrapped lines? The alternative is to actually *prefer* two-space indentation for all list items, which we can actually specify by setting [`start_indented: true`](https://github.com/DavidAnson/markdownlint/blob/main/schema/.markdownlint.yaml#L33-L40). Then MD007 shouldn't complain about the original formatting at all.

Reply:
From your perspective, what do you prefer `start_indented: true` or `start_indented: false`? Our convention so far was the former, so we might as well keep it if you don't feel strongly about this / there are no significant downsides.

Reply:
Considering the convention has been to use two-space indents for lists, I think we should keep that just for simplicity because this is purely a style choice. So I've updated the lint config to prefer two-space indents (beware that this applies only to unordered/bulleted lists; there is no matching config for ordered lists), and I reverted the change that applied the indentation changes.